packages/siopv2-oid4vp-op-auth/src/machine/Siopv2Machine.ts

```diff
@@ -344,10 +367,7 @@ const createSiopv2Machine = (opts: CreateSiopv2MachineOpts): Siopv2StateMachine
       [Siopv2MachineStates.handleError]: {
         id: Siopv2MachineStates.handleError,
         on: {
-          [Siopv2MachineEvents.NEXT]: {
-            target: Siopv2MachineStates.error,
-          },
-          [Siopv2MachineEvents.PREVIOUS]: {
+          '': {
```

Comment:
Why the need for the empty string to begin with? There is an always transition available

Reply:
For what I understand, 'always' is reevaluted / checked continuously even when not in / after a transition.
'' is evaluated only once when the state is entered.
Since I experienced duplicate triggers on the navigationStateListener this seemed the better choice.
Also I don't understand why in the previous situation we only go to error for NEXT/PREVIOUS events. When something fails in a background / unattended task to should also navigate to the error page.